cmd/update-main.go

```diff
@@ -140,6 +141,26 @@ func IsKubernetes() bool {
 	return os.Getenv("KUBERNETES_SERVICE_HOST") != ""
 }
 
+// Minio Helm chart uses DownwardAPIFile to write pod label info to /podinfo/labels
+// More info: https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#store-pod-fields
+// Check if this is Helm package installation and report helm chart version
+func getHelmVersion(helmInfoFilePath string) string {
+	// Check if the file exists. Ignore errors as Minio can be deployed without Helm charts as well.
+	helmInfoFile, err := os.Open(helmInfoFilePath)
+	if err == nil {
+		scanner := bufio.NewScanner(helmInfoFile)
+		for scanner.Scan() {
+			if strings.Contains(scanner.Text(), "chart=") {
+				helmChartVersion := strings.TrimPrefix(scanner.Text(), "chart=")
+				// remove quotes from the chart version
+				helmChartVersion = strings.Trim(helmChartVersion, `"`)
```

Comment:
You don't need to re-assign here.. just do `return strings.Trim(helmChartVersion, `"`)`